frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx

```diff
@@ -496,7 +586,7 @@ function Pipelines({ type }) {
           placement="bottomLeft"
           onOpenChange={() => setSelectedPorD(record)}
         >
-          <EllipsisOutlined rotate={90} className="p-or-d-actions" />
+          <EllipsisOutlined className="p-or-d-actions cur-pointer" />
```

Comment:
why is `rotate={90}` removed?

Reply:
@Deepak-Kesavan I removed it in order to maintain consistency with the API deployment page.